src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  box-sizing: border-box;
+  width: 100%;
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+.search--big {
+  height: 70px;
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-size: 19px;
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  padding-left: 62px;
+}
+
+.search--small {
+  height: 42px;
+  font-size: 14px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-size: 11px;
+  background-repeat: no-repeat;
+  background-position: 13px center;
+  padding: 0 0 0 33px;
+  padding-left: 33px;
+}
+
+.search::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search--big:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-size: 19px;
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  padding-left: 62px;
```

Comment:
please, remove unnecessary properties, if you don`t know what to delete, feel free to ask in the slack chat